index.html

```diff
@@ -140,10 +219,47 @@ <h2 class="tonic">Tonic</h2>
                </ul>
                <button class="see-project">See Project</button>
            </div>
+           </div>
+            </section>
+            <section class="sec4 desktop">
+              <div class="sec-content">
+                <div class="image-div"> 
+                  <img src="images/tonic-desk.jpg" alt="image" class="workplace-image">
+                </div>
+                <div class="sec-cont">
+                <h2 class="tonic desk">Facebook 360</h2>
+                <ul class="set">
+                  <li>
+                    <p class="canopy">FACEBOOK</p> 
+                  </li>
+                  <li class="dot-style">
+                        <img width="10px" height="10px" src="images/Dot.png" alt="dot">
+                       <p class="back">Full Stack Dev</p>
+                      </li>
+                      <li class="dot-style">
+                        <img width="10px" height="10px" src="images/Dot.png" alt="dot">
+                       <p class="year">2015</p>
+                      </li>
+                   </ul>
+                   <p class="daily">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                   </p>
+                   <ul class="list-tech f-margin">
+                       <li>html</li>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>javascript</li>
+                   </ul>
+                   <button class="see-project blue b-margin">See Project</button>
+               </div>
+               </div>
             </section>
-            <section class="sec5">
+            <section class="sec5 mobile">
                 <div class="sec-content">
+                  <div class="image-div"> 
                     <img src="images/professional.png" alt="image" class="workplace-image">
+                  </div>
```

Comment:
- [OPTIONAL] Kindly make sure to fit your project image into the project card correctly in order to meet the Figma design. :wink: 

- [OPTIONAL] Kindly make sure that the `See Project button` looks exactly as the one on the provided Figma design:wink
-
    - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/201432666-f976640a-4ce8-4a78-8234-83980b909c02.png)


    - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/201432721-ca5e732b-7299-4fd6-a680-2fe80ff3a668.png)